examples/next-prisma-starter/src/server/env.js

```diff
@@ -3,7 +3,7 @@
  * This file is included in `/next.config.js` which ensures the app isn't built with invalid env vars.
  * It has to be a `.js`-file to be imported there.
  */
-/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-require-imports */
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/pull/8334: we merged the two rules into the one `no-require-imports`.